composer.json

```diff
@@ -32,7 +32,7 @@
         "icecave/parity": "1.0.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2.2.20||~2.15.1",
+        "friendsofphp/php-cs-fixer": "~2.2.20 || ^2.15.1",
```

Comment:
This was pinned due to an issue with newer minor versions of php-cs-fixer rewriting code in a way that actually caused logic changes, without warning. I would prefer to continue with pinning specific versions if possible, as this happened on multiple occasions with different versions of php-cs-fixer.

Is there are strong reason that requires loosening this constraint?

Reply:
The main reason was allowing it to install fine on PHP 8 in development, because 2.15 requires php 5/7. But I'm happy to pin it to 2.19 instead to avoid surprises in the future.